src/Command/Apply.php

```diff
@@ -101,17 +68,11 @@ public function __construct(
     protected function configure()
     {
         $this->setName(self::NAME)
-            ->setDescription('Apply patches')
+            ->setDescription('Applies patches. The list of patches should pass as a command argument')
             ->addArgument(
-                self::ARG_QUALITY_PATCHES,
-                InputArgument::IS_ARRAY,
-                'List of quality patches to apply'
-            )->addOption(
-                self::OPT_GIT_INSTALLATION,
-                null,
-                InputOption::VALUE_OPTIONAL,
-                'Is git installation',
-                false
+                self::ARG_LIST_OF_PATCHES,
```

Comment:
It's a new command - ./magento-patches apply, 
./ece-patches apply remains without changes

Reply:
@shiftedreality or you mean removing of --git-installation option?
Previously we discussed this with Bohdan and Olexandr, agreed that we can just not call ./ece-patch apply command from  ece-tools in case of git installation